install/ubuntu-installation-guide.md

```diff
@@ -2,62 +2,31 @@
 
 > **Notes:**
 >
-> - Kata Containers packages are available for Ubuntu\* **16.04** and **17.10** (currently `x86_64` only).
+> - Kata Containers packages are available for [Ubuntu\*](https://www.ubuntu.com)
+>   versions **16.04** and **17.10** (currently `x86_64` only).
 >
 > - If you are installing on a system that already has Clear Containers or `runv` installed,
 >   first read [the upgrading document](../Upgrading.md).
 >
-> - This install guide is specific to integration with Docker.  If you want to use Kata
->   Containers through Kubernetes, see the "Developer Guide" [here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#if-you-want-to-run-kata-containers-with-kubernetes).
+> - If you do not wish to copy or type all these instructions by hand, you can use either use the
```

Comment:
Lines 11-14 suggested rewrite:

If you do not want to copy or type all these instructions by hand, you can use the kata-manager script to install the packaged system including your chosen container manager. Alternatively, you can generate a runnable shell script from individual documents using the...